File: CustomerMatterManagementAPI/Data/DTOs/UserDTO.cs
public class UserSignUpDTO
{
    public string Email { get; set; } = null!;
    public string FirstName { get; set; } = null!;
    public string LastName { get; set; } = null!;
    public string Password { get; set; } = null!; // plain text, only for transport
    public string FirmName { get; set; } = null!;
}

File: CustomerMatterManagerUI/src/models/userDto.ts
export interface UserSignUpDTO {
    email: string;
    firstName: string;
    lastName: string;
    password: string;
    firmName: string;
}

export interface UserDetailDTO {
    emailAddr: string;
    firstName: string;
    lastName: string;
    firmName: string;
}

export interface UserLoginDto {
    email: string;
    password: string;
}